services/director-v2/src/simcore_service_director_v2/modules/dynamic_sidecar/docker_service_specs/sidecar.py

```diff
@@ -123,11 +123,18 @@ def _get_environment_variables(
         "SIMCORE_HOST_NAME": scheduler_data.service_name,
         "STORAGE_HOST": app_settings.DIRECTOR_V2_STORAGE.STORAGE_HOST,
         "STORAGE_PORT": f"{app_settings.DIRECTOR_V2_STORAGE.STORAGE_PORT}",
+        "NODE_PORTS_STORAGE_LOGIN": "",
+        "NODE_PORTS_STORAGE_PASSWORD": "",
         "DY_SIDECAR_SERVICE_KEY": scheduler_data.key,
         "DY_SIDECAR_SERVICE_VERSION": scheduler_data.version,
         "DY_SIDECAR_USER_PREFERENCES_PATH": f"{scheduler_data.user_preferences_path}",
         "DY_SIDECAR_PRODUCT_NAME": f"{scheduler_data.product_name}",
         "NODE_PORTS_400_REQUEST_TIMEOUT_ATTEMPTS": f"{app_settings.DIRECTOR_V2_NODE_PORTS_400_REQUEST_TIMEOUT_ATTEMPTS}",
+        "NODE_PORTS_STORAGE_AUTH": (
+            app_settings.DIRECTOR_V2_NODE_PORTS_STORAGE_AUTH.json()
```

Comment:
In releation to the comment above. Here I would have a special function that cleary indicates that the secrets are in the clear! But not "generall"

Reply:
I have changed this and should read better now